staging/src/k8s.io/apiserver/pkg/endpoints/discovery/root.go

```diff
@@ -33,12 +34,21 @@ import (
 // GroupManager is an interface that allows dynamic mutation of the existing webservice to handle
 // API groups being added or removed.
 type GroupManager interface {
```

Comment:
have you double checked that nobody in the ecosystem implements this? It's very unlikely though. Pretty internal code.

Reply:
In the codebase, I found only that we use it as manager `DiscoveryGroupManager discovery.GroupManager`, no other implementations. ecosystem might be harder to check.

Reply:
Internet quick search says - nobody is using it :D